README.md

```diff
@@ -606,6 +607,16 @@ my_text = Text(settings, my_callback)
 my_text.set_text("Some awesome text")
 ```
 
+## Availability Management
+
+This project supports manual availability topics so Home Assistant can reflect whether an entity is online or offline.
+
+- Set `manual_availability=True` in `Settings` to enable manual availability handling.
+- The library will set a retained LWT ("offline"). After your entity is ready, publish "online" to the same topic.
```

Comment:
If you introduce an abbreviation - introduce it first: Last Will and Testament (LWT)